tools/pipelines/build-docs.yml

```diff
@@ -6,11 +6,20 @@
 
 name: $(Build.BuildId)
 
+parameters:
+  - name: deployOverride
```

Comment:
Enables deployment overrides, similar to how we can override publishing packages.